UKHomeOffice/nmsw-ui
# Configure sit environment

Configure SIT environment
Added test for ETA interstitial page [NMSW-1372NMSW UPT help content updates

## To test
Run with cypress runner
```
npm run cypress:open:dev -- --env MAIL_API_KEY=${MAIL_API_KEY}
```
and choose amg-error-validation.feature
check-your-answers-page.feature
or in cmd line
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/landing-page.feature
```
